src/repository/HelpOfferRepository.js

```diff
@@ -33,23 +34,23 @@ class OfferdHelpRepository extends BaseRepository {
     ];
     const user = {
       path: 'user',
-      select: ['photo', 'phone', 'name', 'birthday', 'address.city'],
+      select: commomUserFields,
```

Comment:
Como esse PR é de refatoração e todos os lugares que a gente passa a commonUserFields executam a mesma lógica, será que não poderíamos criar um loop para pegar todos esses dados e guardar num objeto? Algo como o código a seguir
```suggestion
      const userInfo = {user: null, possibleHelpedUsers : null, possibleEntities: null, helpedUsers: null}
      Object.keys(userInfo).forEach(key=> {
          userInfo[key] =  {
               path: key,
               select: commonUserFields,
          }
       })
```

Reply:
Feito